config/fair-queue.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | FairQueue Route Middleware
+    |--------------------------------------------------------------------------
+    |
+    | These middleware will get attached onto each FairQueue route, giving you
+    | the chance to add your own middleware to this list or change any of
+    | the existing middleware. Or, you can simply stick with this list.
+    |
+    */
+    'middleware' => ['web', 'horizon'],
+
+    'host'     => env('FAIR_QUEUE_REDIS_HOST', '127.0.0.1'),
```

Comment:
oh these parameters should not be here. we need a database config section instead.